alternator/server.cc

```diff
@@ -547,22 +547,26 @@ server::server(executor& exec, service::storage_proxy& proxy, gms::gossiper& gos
 }
 
 future<> server::init(net::inet_address addr, std::optional<uint16_t> port, std::optional<uint16_t> https_port, std::optional<tls::credentials_builder> creds,
-        bool enforce_authorization, semaphore* memory_limiter, utils::updateable_value<uint32_t> max_concurrent_requests) {
+        bool enforce_authorization, semaphore* memory_limiter, utils::updateable_value<uint32_t> max_concurrent_requests, int listen_backlogs) {
```

Comment:
nitpick: I think there's no need for the "s" at the end of the variable name, it's not a plural. It can be just "listen_backlog".